src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/TestGenieSettingsStateTest.kt

```diff
@@ -44,7 +44,7 @@ class TestGenieSettingsStateTest {
     private fun valueGenerator(): Stream<Arguments> {
         return Stream.of(
             Arguments.of({ settingsState.minimize = !settingsState.minimize }, mutableListOf("-Dalgorithm=DYNAMOSA", "-Dminimize=false")),
-            Arguments.of({ settingsState.junitCheck = !settingsState.junitCheck }, mutableListOf("-Dalgorithm=DYNAMOSA", "-Djunit_check=true")),
+            Arguments.of({ settingsState.junitCheck = !settingsState.junitCheck }, mutableListOf("-Dalgorithm=DYNAMOSA", "-Djunit_check=false")),
```

Comment:
Why exactly are you setting `junit_check` to false if `DYNAMOSA` is used? Did i miss something in the meeting with the client?

Reply:
`junit_check` is what verifies that all tests in the generated test suite actually pass, afaik.
Because EvoSuite can generate faulty tests, which we don't want to have by default.